sample/src/main/res/values/colors.xml

```diff
@@ -1,15 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources xmlns:tools="http://schemas.android.com/tools">
-    <!--
-    Palette from
-    https://material.io/tools/color/#!/?view.left=0&view.right=0&primary.color=01579B&secondary.color=00E676
```

Comment:
Would be cool if we still have some reference to the Chucker palette somewhere. It doesn't necessary has to be in the `colors.xml`.

Reply:
Ok, I will write it down into the backlog)